README.md

```diff
@@ -344,22 +344,23 @@ If you have received the package and only want to build and run the docker conta
 To run it in a docker container with access to your local network interface, run:
 
 ```bash
-docker run                                                  \
-    --detach                                                \
-    --env         ETHEREUM_PRIVATE_KEY                      \
-    --env         SIGNING_SERVICE_PRIVATE_KEY               \
-    --env         SENTRY_DSN                                \
-    --network     host                                      \
-    --hostname    signing-service                           \
-    --name        signing-service                           \
-    --restart     on-failure                                \
-    concent-signing-service                                 \
-        concent.golem.network                               \
-        85cZzVjahnRpUBwm0zlNnqTdYom1LF1P1WNShLg17cmhN2Us    \
-        --concent-cluster-port                  9055        \
-        --ethereum-private-key-from-env                     \
-        --signing-service-private-key-from-env              \
-        --sentry-dsn-from-env                               \
+docker run                                                                                         \
+    --detach                                                                                       \
+    --env         ETHEREUM_PRIVATE_KEY                                                             \
+    --env         SIGNING_SERVICE_PRIVATE_KEY                                                      \
+    --env         SENTRY_DSN                                                                       \
+    --network     host                                                                             \
+    --hostname    signing-service                                                                  \
+    --name        signing-service                                                                  \
+    --volume      /var/daily_thresholds:/usr/lib/signing_service/signing-service/daily_thresholds  \
```

Comment:
I think that `/var/log/concent/daily_thresholds/` might be a better directory for those files on the host. In the container too, actually, but that would require a change in concent code. The application is not supposed to have write access to its directory in `/usr/lib/`

Reply:
The application is not have access to write in different directory then `/usr/lib/signing_service/signing-service`. All other files and directories are own by root.

Reply:
The path was changed on the host side.